### DISCUSSION TITLE
user notifications

### DISCUSSION THREAD

**Discussion by FrontEnd07** [def-studio/telegraph#469] [Q&A]
I want to implement sending a message to a specific user, or notifying all subscribers about news, what is the best way to achieve this?

**Comment by fabio-ivona** [comment]
Probably keeping track of users that subscribe or send message to your bots, mapping them with your app - [ ] users (or a dedicated model)

**Reply by FrontEnd07** [reply]
How to send messages by a user by its identifier?

**Reply by fabio-ivona** [reply]
You should store it's chatid

**Reply by FrontEnd07** [reply]
I have chats, I just don’t know which method is responsible for sending a message to the chat by ID

**Reply by fabio-ivona** [reply]
`TelegraphChat::find($chat_id)->message('Hi!')->send()`

**Reply by fabio-ivona** [reply]
obv, your chat_id should be present in telegraph_chats tables

**Reply by FrontEnd07** [reply]
If it is possible to make a request in one call and notify everyone on the list, how can I make a general send?

**Reply by fabio-ivona** [reply]
mass messages are not supported, sorry
